spec/factories/overseas_country.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :overseas_country do
-    country_code { "AF" }
+    country_code { "0df4c2e6-74f9-e811-a97a-000d3a2760f2" }
```

Comment:
Ditto for moving these to constants so it's clear what they are

Reply:
yep

Reply:
yep